src/components/BaseModal.vue

```diff
@@ -35,7 +37,7 @@ onBeforeUnmount(() => {
           <slot name="header" />
         </div>
         <div class="modal-body">
-          <slot />
+          <slot :max-height="maxHeight" />
```

Comment:
Do we still need this? And the maxWidth prop we can also remove I think

Reply:
We still need maxHeight slot prop for infinite scroll container, for explicit height. Removed maxWidth prop.